src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.ts

```diff
@@ -102,18 +109,16 @@ export class BootstrapCreateModalComponent extends BaseModal implements OnDestro
           ++checkedCount;
         }
       });
-
+      this.generateBtnDisable = poolsControl.status == 'DISABLED';
```

Comment:
Also, this is checking against the control of all the pools, I believe the idea would be to check if any pool has a valid status and then enable the button